fabric-fluids-v1/src/main/java/net/fabricmc/fabric/access/fluids/BucketItemAccess.java

```diff
@@ -0,0 +1,7 @@
+package net.fabricmc.fabric.access.fluids;
+
+import net.minecraft.fluid.Fluid;
+
+public interface BucketItemAccess {
```

Comment:
If this interface is in api, a `Item -> Optional<BucketDuck>` static method should make sense

Reply:
Access is not the right name. `Access` to me can be easily confused with `Accessor`

Reply:
I will forever reject that naming "do you call your mixins Mixinors?", but fine, just this once 😉

Reply:
it's not an api no